addOns/ascanrules/ascanrules.gradle.kts

```diff
@@ -27,6 +27,7 @@ dependencies {
 
     testImplementation(parent!!.childProjects.get("commonlib")!!)
     testImplementation(parent!!.childProjects.get("commonlib")!!.sourceSets.test.get().output)
+    compile("com.googlecode.concurrent-trees", "concurrent-trees", "2.6.1")
```

Comment:
Can you move compile dependency directly after compileOnly so dependencies are ordered

Reply:
Thx, I will move it.

Reply:
Should be `implementation` (`compile` is superseded).

Reply:
Done :)